functions/venue.js

```diff
@@ -600,12 +612,14 @@ exports.updateVenue = functions.https.onCall(async (data, context) => {
     throw new HttpsError("not-found", `Venue ${venueId} not found`);
   }
 
+  console.log("asdasdasd");
   // @debt this is exactly the same as in updateVenue_v2
   const updated = doc.data();
 
   // @debt refactor function so it doesn't mutate the passed in updated object, but efficiently returns an updated one instead
   createBaseUpdateVenueData(data, updated);
 
+  console.log(updated);
```

Comment:
```suggestion
```

Reply:
Removed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0e1ea16c81870c898660c5e5b60aa9a143b57c80